src/Microsoft.AspNet.Identity/RoleManager.cs

```diff
@@ -32,6 +33,7 @@ public class RoleManager<TRole> : IDisposable where TRole : class
                 throw new ArgumentNullException("store");
             }
             Store = store;
+            KeyNormalizer = keyNormalizer ?? new UpperInvariantKeyNormalizer();
```

Comment:
Can you please ping the engineering team about this pattern? Not blocking on it.